samtranslator/model/sam_resources.py

```diff
@@ -1197,6 +1198,7 @@ class SamApi(SamResourceMacro):
     Mode: Optional[Intrinsicable[str]]
     DisableExecuteApiEndpoint: Optional[Intrinsicable[bool]]
     ApiKeySourceType: Optional[Intrinsicable[str]]
+    AlwaysDeploy: Optional[bool]
```

Comment:
Are we going to support `AlwaysDeploy` as a global?

Reply:
I was originally thinking no since it's more of a workaround, but it doesn't hurt. I've added it.